include/wlr/types/wlr_keyboard_group.h

```diff
@@ -18,6 +18,30 @@ struct wlr_keyboard_group {
 	struct wlr_input_device *input_device;
 	struct wl_list devices; // keyboard_group_device::link
 	struct wl_list keys; // keyboard_group_key::link
+
+	struct {
+		/*
+		 * Sent when a keyboard has entered the group with keys currently
+		 * pressed that are not pressed by any other keyboard in the group. The
+		 * data for this signal will be a wl_array containing the key codes.
+		 * This should be used to update the compositor's internal state.
+		 * Bindings should not be triggered based off of these key codes and
+		 * they should also caused any surfaces to be notified.
```

Comment:
Small typo in this last sentence, I think.

Reply:
Oops. Fixed